pom.xml

```diff
@@ -172,7 +172,7 @@
 		<log4j.version>2.17.1</log4j.version>
 		<netty.version>4.1.66.Final</netty.version>
 		<protobuf-java.version>3.19.4</protobuf-java.version>
-		<swirlds.version>0.23.0</swirlds.version>
+		<swirlds.version>0.22.1</swirlds.version>
```

Comment:
This shouldn't be necessary anymore and we shouldn't commit this change, to make 0.23 work, you need to add this line to your hedera-node/settings.txt to start locally now:
loadKeysFromPfxFiles,                          0